ui/selectors/selectors.js

```diff
@@ -2021,14 +2008,13 @@ export function getIsNetworkSupportedByBlockaid(state) {
   return isSupported;
 }
 
-export const getAllEnabledNetworks = createDeepEqualSelector(
-  getNonTestNetworks,
-  getAllNetworks,
-  getShowTestNetworks,
-  (nonTestNetworks, allNetworks, showTestnetNetworks) => {
-    return showTestnetNetworks ? allNetworks : nonTestNetworks;
-  },
-);
+export function getAllEnabledNetworks(state) {
+  const nonTestNetworks = getNonTestNetworks(state);
+  const allNetworks = getAllNetworks(state);
+  const showTestnetNetworks = getShowTestNetworks(state);
+
+  return showTestnetNetworks ? allNetworks : nonTestNetworks;
+}
```

Comment:
Can I ask why we made this change?

Reply:
Thank you @darkwing ! my bad, must have happened during merge 🙏